### PR TITLE
[jsk_spot_startup] update joy configurations and documentation, add R1 support.

### PR DESCRIPTION
**Pull request by sktometometo**
- Add R1 joy pad support
- Update joy configuration
  - add a stair mode button to dualshock3 mapping
- Update documentation
  - update the mapping picture and add R1 documentation

# R1 teleop test 
https://user-images.githubusercontent.com/9410362/114703836-cdcb3500-9d60-11eb-8b04-0be0fa4cac1f.mp4

# Key Mapping for DualShock3
SpotTeleopDualshock3

# Key Mapping for Magicsee R1
SpotTeleopR1

### CODE REVIEW

**Comment by sktometometo** [comment]
confirmed that this works well

**Comment by mqcmd196** [comment]
Cool! I'll see it next week.

File: jsk_spot_robot/jsk_spot_startup/launch/jsk_spot_bringup.launch [hunk 1]
```diff
@@ -30,5 +32,12 @@
   <!-- teleop node -->
   <include
       file="$(find jsk_spot_teleop)/launch/teleop.launch"
-      />
+      >
+      <arg name="pad_type" value="R1" />
+  </include>
+  <include
```
Comment (tongtybj) [inline jsk_spot_robot/jsk_spot_startup/launch/jsk_spot_bringup.launch:38]: Do we need to use `<if>` and `<unless>` for these <include> blocks? It seems like that the current code will simultaneously create two teleop.launch.

File: jsk_spot_robot/jsk_spot_startup/launch/jsk_spot_bringup.launch [hunk 2]
```diff
@@ -3,6 +3,8 @@
   <arg name="password" default="dummypassword" />
   <arg name="hostname" default="192.168.50.3" />
 
+  <arg name="pad_type" default="R1" /> <!-- Option: dualshock3, R1 -->
```
Comment (tongtybj) [inline jsk_spot_robot/jsk_spot_startup/launch/jsk_spot_bringup.launch:6]: this arg is not used?

**Comment by tongtybj** [comment]
Grateful for you! Amazing teleop system!

File: jsk_spot_robot/jsk_spot_startup/launch/jsk_spot_bringup.launch (hunk 2 above)
Comment (sktometometo) [reply jsk_spot_robot/jsk_spot_startup/launch/jsk_spot_bringup.launch:6]: No, not used. I notice it is redundant. thanks.

File: jsk_spot_robot/jsk_spot_startup/launch/jsk_spot_bringup.launch (hunk 1 above)
Comment (sktometometo) [reply jsk_spot_robot/jsk_spot_startup/launch/jsk_spot_bringup.launch:38]: that is intended behavior. I think both pads have pros. and cons. But if it is required to stop and launch ’jsk_spot_bringup.launch’ when switching a controller, it is inconvenient. So this PR uses both controllers simultaneously.

Maybe it depends on use cases. Please tell me if there is any suggestion.

**Review by tongtybj — APPROVED** [review]

**Pull request merged**